template/app/src/main/java/com/xmartlabs/template/helper/EmptyOnPageChangeListener.kt

```diff
@@ -0,0 +1,9 @@
+package com.xmartlabs.template.helper
+
+import android.support.v4.view.ViewPager
+
+open class EmptyOnPageChangeListener : ViewPager.OnPageChangeListener {
```

Comment:
I'd like empty more because we use base when we actually add code. What do you think?